wd_helper.au3

```diff
@@ -1783,33 +1783,24 @@ Func _WD_ElementActionEx($sSession, $sElement, $sCommand, $iXOffset = Default, $
 
 		Case 'doubleclick'
 			$sPostHoverAction = _
-					',{"button": ' & $iButton & ', "type": "pointerDown"}' & _
-					',{"button": ' & $iButton & ', "type": "pointerUp"}' & _
-					',{"button": ' & $iButton & ', "type": "pointerDown"}' & _
-					',{"button": ' & $iButton & ', "type": "pointerUp"}'
+					__WD_JsonButtonAction($iButton, "pointerDown") & _
+					__WD_JsonButtonAction($iButton, "pointerUp") & _
+					__WD_JsonButtonAction($iButton, "pointerDown") & _
+					__WD_JsonButtonAction($iButton, "pointerUp")
 
 		Case 'rightclick'
 			$sPostHoverAction = _
-					',{' & _
-					'	"button": 2' & _
-					'	,"type": "pointerDown"}' & _
-					',{' & _
-					'	"button": 2' & _
-					'	,"type": "pointerUp"' & _
-					'}'
+					__WD_JsonButtonAction("2", "pointerDown") & _
+					__WD_JsonButtonAction("2", "pointerUp")
```

Comment:
First parameter is defined as an integer, not a string. 😉

Reply:
Fixed